.github/release_workflow.md

```diff
@@ -1,10 +1,13 @@
 # Release workflow
 
-This file contains the workflow required to make a `PyBaMM` release on GitHub, PyPI, and conda-forge by the maintainers.
+This file contains the workflow required to make a `PyBaMM` release on
+GitHub, PyPI, and conda-forge by the maintainers.
 
-## rc0 releases (automated)
+## Initial release (automated)
 
-1. The `update_version.yml` workflow will run on every 1st of January, May and September, updating incrementing the version to `vYY.MMrc0` by running `scripts/update_version.py` in the following files -
+1. The `update_version.yml` workflow will run on every 1st of January, May
+   and September, updating incrementing the version to `vYY.MM.0` by running
```

Comment:
Are we moving away from the concept of release candidates? The "rc" term is a PyPI and PEP thing. "rc" allows PyPI to identify if it is a final or a pre release.

Reply:
If we are, then we don't need this entire infrastructure. We can just do normal releases with bug fixes whenever we want to, just how it was before.

Reply:
I like the idea of the initial release branch being created automatically. I agree that manually triggering the workflow is a bit useless. I can remove that.

Yeah in general I thought it would be good to move away from the release candidates. It seems to make our process take a month for each release. Late releases are not the end of the world when there is a lot of time between them, but it would be nicer to get some of our changes out faster.